.github/workflows/conda-lock.yml

```diff
@@ -49,7 +50,7 @@ jobs:
           path: ci/conda-lock/*/${{ matrix.python-version }}.lock
 
   condalock_pr:
-    if: github.event_name == 'schedule' || github.event_name == 'workflow_dispatch'
+    if: (github.repository_owner == 'ibis-project') && (github.event_name == 'schedule' || github.event_name == 'workflow_dispatch')
```

Comment:
double check me here, I think we won't know if this fails until the cron fires.

Reply:
That's true, but there's nothing we can do about it AFAIK.